store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/operation/AbstractAccumuloTwoSetSeededOperation.java

```diff
@@ -48,7 +47,6 @@ public AbstractAccumuloTwoSetSeededOperation(final Iterable<SEED_TYPE> seedsA, f
         this.setSeedsB(seedsB);
     }
 
-    @JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.WRAPPER_OBJECT)
     @JsonGetter(value = "seedsB")
```

Comment:
I think it should be the same annotation as seeds in AbstractGetOperation:
@JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.PROPERTY, property = "class")